backfill/2022-04-27-firefox-desktop-glean-rejected/README.md

```diff
@@ -0,0 +1,177 @@
+
+```bash
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_stable.baseline_v1 moz-fx-data-shared-prod:firefox_desktop_stable.baseline_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_stable.deletion_request_v1 moz-fx-data-shared-prod:firefox_desktop_stable.deletion_request_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_stable.events_v1 moz-fx-data-shared-prod:firefox_desktop_stable.events_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_stable.fog_validation_v1 moz-fx-data-shared-prod:firefox_desktop_stable.fog_validation_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_stable.metrics_v1 moz-fx-data-shared-prod:firefox_desktop_stable.metrics_v1
+```
+
+For the current day, we have to append into live tables instead:
+
+```bash
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_live.baseline_v1'$20220427' moz-fx-data-shared-prod:firefox_desktop_live.baseline_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_live.deletion_request_v1'$20220427' moz-fx-data-shared-prod:firefox_desktop_live.deletion_request_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_live.events_v1'$20220427' moz-fx-data-shared-prod:firefox_desktop_live.events_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_live.fog_validation_v1'$20220427' moz-fx-data-shared-prod:firefox_desktop_live.fog_validation_v1
+bq cp --append_table moz-fx-data-backfill-10:firefox_desktop_live.metrics_v1'$20220427' moz-fx-data-shared-prod:firefox_desktop_live.metrics_v1
+```
+
+## Step 5:  Clean up
+
+1.  DE will remove any GCS buckets created by Dataflow jobs:
+
+    ```bash
+    # BE CAREFUL!  This removes all GCS buckets from the target project.  It cannot be undone.
+
+    gsutil ls -p moz-fx-data-backfill-10 | xargs gsutil -m rm -r
+    ```
+
+    Log:
+
+    ```
+    ...
+    Operation completed over 250 objects.
+    Removing gs://dataflow-staging-us-central1-339436751542/...
+    ```
+
+2.  Data SRE will delete the errors for the payloads that have now been successfully ingested:
+
+```sql
+DELETE FROM `moz-fx-data-shared-prod.payload_bytes_error.structured`
+WHERE
+  DATE(submission_timestamp) BETWEEN '2022-04-12' AND '2022-04-27'
+  AND document_namespace = 'firefox-desktop'
+  AND error_message LIKE 'com.mozilla.telemetry.decoder.MessageScrubber$UnwantedDataException: 1684980'
+```
+
+This in practice produced an error message:
+
+> UPDATE or DELETE statement over table moz-fx-data-shared-prod.payload_bytes_error.structured would affect rows in the streaming buffer, which is not supported
+
+We modified this to not affect the current day (made the end date 2022-04-26) to get around the problem. This deleted 8,023,263 rows.
+
+We'll need to circle back to this after 2022-04-27 is complete to delete the final day:
```

Comment:
Minor markdown formatting issue that subsequent lines for each ordered list item need to be indented to be included within that list item (see item 1 for an example).

As it is, items 3 & 4 are technically separate ordered lists, but luckily GitHub's markdown parsing respects the numbering (e.g. item 3 renders as `<ol start="3">`), so this is more a cosmetic issue.